docs/spark/tutorials/get-started-macos-instructions.md

```diff
@@ -0,0 +1,126 @@
+  - Select a **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release from .NET for Apache Spark GitHub Releases page and download into your local machine (e.g., `/bin/Microsoft.Spark.Worker/`).
+  - **IMPORTANT** Create a new environment variable using ```export DOTNET_WORKER_DIR <your_path>``` and set it to the directory where you downloaded and extracted the Microsoft.Spark.Worker (e.g., `/bin/Microsoft.Spark.Worker/`).
+
+## Authoring a .NET for Apache Spark App
+
+- Use the `dotnet` CLI to create a console application.
+
+    ```
+    dotnet new console -o HelloSpark
+    ```
+    
+- Install `Microsoft.Spark` Nuget package into the project from the [spark nuget.org feed](https://www.nuget.org/profiles/spark) - see [Ways to install Nuget Package](https://docs.microsoft.com/en-us/nuget/consume-packages/ways-to-install-a-package)
+    
+    ```
+    cd HelloSpark
+    dotnet add package Microsoft.Spark
+    ```
+    
+- Replace the contents of the `Program.cs` file with the following code:
+    
+    ```csharp
+    using Microsoft.Spark.Sql;
+
+    namespace HelloSpark
+    {
+        class Program
+        {
+            static void Main(string[] args)
+            {
+                var spark = SparkSession.Builder().GetOrCreate();
+                var df = spark.Read().Json("people.json");
+                df.Show();
+            }
+        }
+    }
+    ```
+    
+- Use the `dotnet` CLI to build the application:
+    
+    ```bash
+    dotnet build
+    ```
+
+## Running your .NET for Apache Spark App
+
+- Open your terminal and navigate into your app folder:
+    
+    ```bash
+    cd <your-app-output-directory>
+    ```
+    
+- Create `people.json` with the following content:
+    
+    ```json
+    { "name" : "Michael" }
+    { "name" : "Andy", "age" : 30 }
+    { "name" : "Justin", "age" : 19 }
+    ```
+    
+- Run your app
+    
+    ```bash
+    spark-submit \
+    --class org.apache.spark.deploy.dotnet.DotnetRunner \
+    --master local \
+    microsoft-spark-2.4.x-<version>.jar \
+    dotnet HelloSpark.dll 
+    ```
+    
+    **Note**: This command assumes you have downloaded Apache Spark and added it to your PATH environment variable to be able to use `spark-submit`, otherwise, you would have to use the full path (e.g., `~/spark/bin/spark-submit`).
+    
+- The output of the application should look similar to the output below:
+    
+    ```text
+    +----+-------+
+    | age|   name|
+    +----+-------+
+    |null|Michael|
+    |  30|   Andy|
+    |  19| Justin|
+    +----+-------+
+    ```
+   
+## Next steps
+
+In this tutorial, you learn how to run your .NET for Apache Spark on MacOSX. Continue to learn more about how to run your .NET for Apache Spark on Windows and Ubuntu.
+> [!div class="nextstepaction"]
+
+> [Tutorial: Get Started with .NET for Apache Spark on Ubuntu](get-started-windows-instructions.md)
```

Comment:
```suggestion
> [!div class="nextstepaction"]
> [Tutorial: Get Started with .NET for Apache Spark on Windows](get-started-windows-instructions.md)
```

Reply:
Thanks @gewarren and @Youssef1313 for your comments. I have resolved all the comments accordingly. Thanks!